src/ConvertUTF.h

```diff
@@ -138,6 +138,8 @@
 
 #pragma once
 
+#include <stdbool.h>
```

Comment:
This file is imported from the LLVM project (https://github.com/llvm/llvm-project/blob/master/llvm/include/llvm/Support/ConvertUTF.h). We probably don't to make changes to it if we can help it, unless it's something that's in the upstream.

To be honest this file could probably be moved into the 3rdparty repo to make it more explicit, but that's a separate conversation.

Reply:
OK, patch dropped.